asm/instruction.go

```diff
@@ -60,6 +60,13 @@ func (ins *Instruction) Unmarshal(r io.Reader, bo binary.ByteOrder) (uint64, err
 	}
 
 	ins.Offset = int16(bo.Uint16(data[2:4]))
+
+	// For ALU Ops, the lower 8 bits of the offset field are part of the opcode
+	// within our ASM representation.
+	if ins.OpCode.Class().IsALU() {
```

Comment:
I think I'd be more explicit here: if we're dealing with Mod or Div, check whether offset is 1. If it is change the opcode. Otherwise we may end up importing